### DISCUSSION TITLE
Support for Mongoose 7 / MongoDB NodeJS Driver 5?

### DISCUSSION THREAD

**Discussion by KeithGillette** [typegoose/typegoose#795] [Ideas]
What are the plans to update Typegoose to support the [recent release of Mongoose 7](https://github.com/Automattic/mongoose/blob/master/CHANGELOG.md#700--2023-02-27), which updates to [MongoDB NodeJS Driver 5](https://github.com/mongodb/node-mongodb-native/blob/main/HISTORY.md#500-2023-01-31), both of which include breaking changes?

**Comment by hasezoey** [comment]
Typegoose [v11.0.0-beta.1](https://github.com/typegoose/typegoose/releases/tag/v11.0.0-beta.1) is already out, though may not contain final breaking changes, currently includes mongoose 7 usage
